src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.heading {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.linkForArticle a {
+  color: green;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
name classes using kebab-case